Automattic/jetpack
# Boost: Fix getting-started tracks

Fixes Automattic/boost-cloud#231

## Proposed changes:
* Add two new events for tracking `using_existing_connection`, and `established_connection`
* Wait until connection to dispatch CTA events

### Other information:


## Jetpack product discussion
p1686131295122349-slack-C016BBAFHHS

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Test both,  with a site that is jetpack connected before installing boost and with one that isn't connected
* Make sure appropriate event between `using_existing_connection`, and `established_connection` is firing
* Make sure the CTA events are firing
